person.rb

```diff
@@ -0,0 +1,44 @@
+require '/decorate'
+
+# parent class Person.new(age, name, parent_permision)
+class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def can_use_services?
+    if of_age? || @parent_permission
+      true
+    else
+      false
+    end
+  end
+
+  def correct_name
+    @name
+  end
```

Comment:
- [ ] Please add a method for adding rentals here in the `person` class you did in the book class so any class that inherits it like the teacher or student class can add a new rental.
[Hint] you can add the same method you added in the book class:
```ruby
 def add_rental(date, person)
    Rental.new(date, person, self)
  end
```